python/GafferSceneUI/PathFilterUI.py

```diff
@@ -188,19 +214,28 @@ def __dataMenu( self, vectorDataWidget, menuDefinition ) :
 
 def __popupMenu( menuDefinition, plugValueWidget ) :
 
-	if not isinstance( plugValueWidget, GafferUI.VectorDataPlugValueWidget ) :
-		return
+	selection = None
 
 	plug = plugValueWidget.getPlug()
 	if plug is None:
 		return
-	
-	cellPlug = plug.ancestor( Gaffer.Spreadsheet.CellPlug )
 
-	if cellPlug is None :
-		return
+	node = plug.node()
 
-	selection = IECore.PathMatcher( plugValueWidget.vectorDataWidget().getData()[0] )
+	if isinstance( node, Gaffer.Spreadsheet ) :
+
+		if isinstance( plugValueWidget, GafferUI.VectorDataPlugValueWidget ) :
+			cellPlug = plug.ancestor( Gaffer.Spreadsheet.CellPlug )
+			if cellPlug is None :
+				return
+
+			selection = IECore.PathMatcher( plugValueWidget.vectorDataWidget().getData()[0] )
+
+		elif plug.ancestor( Gaffer.Spreadsheet.RowPlug ) and "scene:path" in node["selector"].getValue() :
+			selection = IECore.PathMatcher( [ plugValueWidget.getPlug().getValue() ] )
```

Comment:
Should scope a context before calling `getValue()`.

Reply:
Fixed in 81faa75d5afc6b55d8164fb008339dbd18db67ea